proto/Cargo.toml

```diff
@@ -13,6 +13,12 @@ description = """
     tendermint-proto is a the Rust implementation of the Tendermint proto structs.
     """
 
+[features]
+default = []
+grpc = ["grpc-server", "grpc-client"]
+grpc-server = ["tonic"]
+grpc-client = ["tonic"]
```

Comment:
As this feature is not used by generated code, perhaps let's not introduce it at this point.